### PR TITLE
add more detailed use examples

### PR DESCRIPTION
**Pull request by komish**
This PR just adds a docs file RECIPES.md that discusses a few ways to use preflight (Binary, Podman, Kubernetes) with manifests and command invocations to use as a starting point.

Signed-off-by: Jose R. Gonzalez <josegonzalez89@gmail.com>

### CODE REVIEW

**Review by acornett21 — APPROVED** [review]

**Review by samira-barouti — APPROVED** [review]
/lgtm

File: docs/RECIPES.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+(e.g. `preflight check operator <bundle`>), but the same concepts can be applied
+to the Container policy (give or take a few configuration changes).
+
+You will also need an index image containing your bundle for each of these approaches.
+See [DOCS](BUILDING_AN_INDEX.md)
+
+## As a Binary on Your Workstation
+
+To run `preflight` on your workstation, you'll first need to download and
+[install](../README.md#Installation)
```
Comment (bcrochet) [inline docs/RECIPES.md:16]: Why is this wrapped weird?

File: docs/RECIPES.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+the binary to your path.
+
+You will also need:
+
+- Your bundle image published to a container registry,
+- An Index Image containing your operator's new bundle published to a container
+  registry.
+- A Kubeconfig for a user with cluster-admin privileges to an OpenShift 4.5+
```
Comment (bcrochet) [inline docs/RECIPES.md:24]: Is this true? Does it have to be cluster-admin? My testing has shown that to not be the case.

File: docs/RECIPES.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  quay.io/opdev/preflight:stable check operator registry.example.org/your-namespace/your-bundle-image:sometag
+```
+
+## As a Job In OpenShift (or Kubernetes)
+
+You should be able to run `preflight` as a job in OpenShift without requiring
+additional privileges or security context constraints.
+
+As with previous examples, you will need to provide `preflight`a Kubeconfig
+mapping a user with cluster-admin privileges to an OpenShift cluster that can be
```
Comment (bcrochet) [inline docs/RECIPES.md:74]: Same as above. I created a standard SA, with a rolebinding to the 'edit' role, then passed that in as PFLT_SERVICEACCOUNT, and a namespace via PFLT_NAMESPACE.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Couple of comments.

File: docs/RECIPES.md (hunk 1 above)
Comment (komish) [reply docs/RECIPES.md:16]: Markdown linting was likely complaining about line length. When rendered, they should collapse to a single line.

File: docs/RECIPES.md (hunk 2 above)
Comment (komish) [reply docs/RECIPES.md:24]: I'll address this in the other comment related to the same topic 👍

File: docs/RECIPES.md (hunk 3 above)
Comment (bcrochet) [reply docs/RECIPES.md:74]: Actually, no. I think that may be why it was failing for me. I hadn't circled back to see why it was failing.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**